test/functional/apps/discover/_runtime_fields_editor.ts

```diff
@@ -39,6 +39,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.common.navigateToApp('discover');
     });
 
+    after(async () => {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover.json');
+    });
+
```

Comment:
This test creates new saved objects that won't be removed by the `unload` call.  We either need to use `clean()` or a custom space.

Reply:
Ahhh, dagnabit, I thought I caught that.  Ok, I'll fixup boss!